src/main/cliapp/src/components/ListTableCell.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 
-export const ListTableCell = ({ template, listData, showBullets }) => {
+export const ListTableCell = ({ template, listData, showBullets, style}) => {
+
+	const leftAlign = { textAlign: 'left' };
+	let alignment = style ?? leftAlign;
 	let suppress = 'list-none'
 	if(showBullets)
 		suppress = ''
 	return (
 		<ul className={`pl-0 ${suppress}`}>
 			{listData?.map((item, index) =>
-				<li key={index}>
+				<li key={index} style={alignment}>
```

Comment:
We don't need this code... we should no hardcode styles.

Reply:
Plus this is going to change ALL ListTableCell usage... the code needs to be on the ListTableCell component not in here, styling that is.

Reply:
This is a single line change:
```<ListTableCell template={listTemplate} listData={sortedSynonyms}/>```
change to:
 ```<ListTableCell styleClass="align-items-start" template={listTemplate} listData={sortedSynonyms}/>```